src/org/labkey/test/components/ui/entities/ParentEntityEditPanel.java

```diff
@@ -101,6 +101,14 @@ private void clickButtonWaitForPanel(WebElement button, int wait)
         Locator infoPanel = Locator.tagWithClass("div", "panel-info");
         int infoCount = infoPanel.findElements(getDriver()).size();
 
+        if(infoCount > 1)
+        {
+            getWrapper().log("Whoa, there appears to be more than one panel in edit mode. That is odd.");
+        }
```

Comment:
Consider using the `TestLogger` to log a warning.
```suggestion
        if(infoCount > 1)
        {
            TestLogger.warn("Whoa, there appears to be more than one panel in edit mode. That is odd.");
        }
```
Has this ever happened? Should it just be an error?

Reply:
Long ago it was possible to have multiple panels open for edit at the same time, however that has changed.
I've updated the code to assert that there is only one panel in edit mode.